packages/gatsby-kontent-components/src/image-element/index.tsx

```diff
@@ -8,7 +8,7 @@ import {
 export interface ImageItem {
   image_id: string;
   url: string;
-  description?: string;
+  description?: string | null;
```

Comment:
so do we need `?` here?

Reply:
description is null when it comes straight from REST DELIVERY API. The others are there due to differences in AssetValueElement and RichTextImage